runtime_lib/test_lib/CMakeLists.txt

```diff
@@ -31,4 +31,32 @@ install(TARGETS test_lib
     PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_PREFIX}/runtime_lib/${AIE_RUNTIME_TARGET}/test_lib/include
 )
 
-install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/test_library.cpp ${CMAKE_CURRENT_SOURCE_DIR}/test_library.h DESTINATION ${CMAKE_INSTALL_PREFIX}/runtime_lib/${AIE_RUNTIME_TARGET}/test_lib)
\ No newline at end of file
+install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/test_library.cpp ${CMAKE_CURRENT_SOURCE_DIR}/test_library.h DESTINATION ${CMAKE_INSTALL_PREFIX}/runtime_lib/${AIE_RUNTIME_TARGET}/test_lib)
+
```

Comment:
Hrm.. does seem odd that runtime_lib/test_lib is compiling test_library.cpp for each host_target, but the file lives and is used in aie_runtime_lib